core/tests/protractor_utils/ExplorationEditorPage.js

```diff
@@ -40,42 +40,39 @@ var ExplorationPlayerPage = require(
 
 var ExplorationEditorPage = function() {
   /*
-   * Components
-   */
-  this.getImprovementsTab = function() {
-    return (
-      new ExplorationEditorImprovementsTab.ExplorationEditorImprovementsTab());
-  };
-  this.getFeedbackTab = function() {
-    return new ExplorationEditorFeedbackTab.ExplorationEditorFeedbackTab();
-  };
-  this.getHistoryTab = function() {
-    return new ExplorationEditorHistoryTab.ExplorationEditorHistoryTab();
-  };
-  this.getMainTab = function() {
-    return new ExplorationEditorMainTab.ExplorationEditorMainTab();
-  };
-  this.getSettingsTab = function() {
-    return new ExplorationEditorSettingsTab.ExplorationEditorSettingsTab();
-  };
-  this.getStatsTab = function() {
-    return new ExplorationEditorStatsTab.ExplorationEditorStatsTab();
-  };
-  this.getTranslationTab = function() {
-    return new ExplorationEditorTranslationTab
-      .ExplorationEditorTranslationTab();
-  };
-  this.getPreviewTab = function() {
-    return new ExplorationPlayerPage.ExplorationPlayerPage();
-  };
-
-  /*
-   * Interactive elements
-   */
+  * Interactive elements
+  */
   var commitMessageInput = element(
     by.css('.protractor-test-commit-message-input'));
   var neutralElement = element.all(by.css('.protractor-test-neutral-element'))
     .first();
+  var expTitle = element(by.css(
+    '.protractor-test-exploration-title-input'));
+  var expObjective = element(by.css(
+    '.protractor-test-exploration-objective-input'));
+  var expTags = element(by.css('.protractor-test-tags'));
+  var expInput = expTags.element(by.tagName('input'));
+  var expCategory = element(
+    by.css('.protractor-test-exploration-category-dropdown'));
+  var expLanguage = element(
+    by.css('.protractor-test-exploration-language-select'));
+  var explorationMetaDataModalHeaderElement = element(
```

Comment:
"metadata" is one word, so the variable should be:

```suggestion
  var explorationMetadataModalHeaderElement = element(
```

Reply:
Fixed!